src/core/style/_variables.scss

```diff
@@ -3,6 +3,10 @@
 
 // Typography
 $md-body-font-size-base: rem(1.400) !default;
+$font-family: Roboto, 'Helvetica Neue', sans-serif !default;
+
+// Media queries
+$md-mobile-screen: 600px;
```

Comment:
According to spec, this would be the breakpoint for a "large handset" or an "xsmall window". Which do you think would be clearer?

https://www.google.com/design/spec/layout/responsive-ui.html#responsive-ui-breakpoints